types/ckeditor__ckeditor5-core/src/commandcollection.d.ts

```diff
@@ -1,18 +1,50 @@
-import Command, { CommandInterface } from './command';
+import Command from './command';
 
 // tslint:disable-next-line:no-empty-interface
 export interface Commands {}
 
-export default class CommandCollection implements Iterable<[string, Command]> {
-    constructor();
-    [Symbol.iterator](): Iterator<[string, Command]>;
+/**
+ * Collection of commands. Its instance is available in {@link module:core/editor/editor~Editor#commands `editor.commands`}.
+ */
+export default class CommandCollection {
```

Comment:
Is this no longer iterable?

Reply:
It is, and the methods are there, but I forgot to make it more clear. It's fixed now.